extension/src/test/Documentation.test.ts

```diff
@@ -0,0 +1,154 @@
+import * as assert from "assert";
+import * as path from "path";
+import * as fs from "fs";
+
+import * as Documentation from "../Documentation";
+import * as SettingsLoader from "../Settings/SettingsLoader";
+import * as Common from "../Common";
+
+suite("Documentation Tests", async function () {
+  const WORKFLOW = process.env.GITHUB_ACTION; // Only run in GitHub Workflow
+  const settings = SettingsLoader.getSettings();
+  const appManifest = SettingsLoader.getAppManifest();
+  const testAppPath = path.join(__dirname, "../../../test-app/Xliff-test");
+  const testAppDocsPath = path.join(testAppPath, settings.docsRootPath);
+  const tempDocsPath = path.join(__dirname, "temp/docs");
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const appPackage = require("../../package.json");
+
+  test("Documentation.generateExternalDocumentation", async function () {
+    if (!WORKFLOW) {
+      this.skip();
+    }
+    this.timeout(10000);
+    // remove docs directory
+    fs.rmdirSync(tempDocsPath, { recursive: true });
+    settings.docsRootPath = tempDocsPath;
+    settings.tooltipDocsFilePath = path.join(
+      tempDocsPath,
+      settings.tooltipDocsFilePath
+    );
+    settings.generateTooltipDocsWithExternalDocs = true;
+    settings.ignoreTransUnitInGeneratedDocumentation = [];
+    settings.tooltipDocsIgnorePageIds = [];
```

Comment:
@jwikman any other settings that should be handled here?

Reply:
Isn't there more ignore-settings?

Reply:
found two more :)

Reply:
- TooltipDocsIgnorePageExtensionIds
- DocsIgnorePaths

Reply:
Haha, I'm a bit slow on the phone... 😃
😴